src/oom/test-oomd-util.c

```diff
@@ -255,32 +261,34 @@ static void test_oomd_pressure_above(void) {
         assert_se(store_loadavg_fixed_point(99, 99, &(ctx[0].memory_pressure.avg60)) == 0);
         assert_se(store_loadavg_fixed_point(99, 99, &(ctx[0].memory_pressure.avg300)) == 0);
         ctx[0].mem_pressure_limit = threshold;
+        ctx[0].mem_pressure_duration_usec = 0;
```

Comment:
should be USEC_INFNITY i guess?

Reply:
So for this, these both need to be 0 since these are testing oomd_pressure_above function which needs to check `diff >= ctx->mem_pressure_duration_usec`. And this function needs to be tested with `ctx->mem_pressure_limit_hit_start` set and not. And we obviously don't want to set this to nonzero because then the test depends on timing

I'll add comment why these should be 0